documentation/using_plugins.md

```diff
@@ -78,6 +78,13 @@ soon as the inventory file is loaded, whereas reference plugins under data keys
 such as `config` or `facts` are resolved once Bolt starts running an action on
 that target.
 
+While plugins are supported for configuration files, they're disabled for some settings. You can see
+which options are available on configuration references pages for
+[bolt.yaml](bolt_configuration_reference.md), [bolt-project.yaml](bolt_project_reference.md), and
```

Comment:
Do we want bolt.yaml here? My vote would be to leave it out cos it's one less thing to whackamole when we hit 3.0

Reply:
Great point, removed it!